src/structures/Presence.js

```diff
@@ -52,11 +62,20 @@ class Presence {
 
     const activity = data.game || data.activity;
     /**
-     * The activity of the presence
+     * The activity of this presence
      * @type {?Activity}
      */
     this.activity = activity ? new Activity(this, activity) : null;
 
+    /**
+     * The devices this presence is on
+     * @type {?object}
+     * @property {string} web
+     * @property {string} mobile
+     * @property {string} desktop
```

Comment:
Same as in the typings.
Or just a general note referencing `Presence#status`, as we do not have a dedicated typedef.